.readme_template.md

```diff
@@ -0,0 +1,114 @@
+
```

Comment:
If we are not using the readme template from the template repo, there is little value in having the readme generated at all. It would be easier to just write the Readme directly.